spec/unit/renderer_spec.rb

```diff
@@ -54,4 +55,10 @@
       }.to raise_error(Dry::View::Renderer::TemplateNotFoundError, /_missing_partial/)
     end
   end
+
+  describe '#chdir' do
+    it 'copies options to new renderer instance' do
+      expect(renderer.chdir('nested').options[:default_encoding]).to eq('utf-8')
```

Comment:
Instead of checking that a single option is the same, could we check that the whole options hash is the same? This expresses the intention better, I think, and should also cover us better for regressions.

Reply:
Yep, you are right. Forced a push with the change.